scapy/layers/tls/tools.py

```diff
@@ -19,10 +23,27 @@ class TLSPlaintext(Packet):
     name = "TLS Plaintext"
     fields_desc = [ ByteEnumField("type", None, _tls_type),
                     ShortEnumField("version", None, _tls_version),
-                    FieldLenField("len", None, length_of="fragment",
-                                  fmt="!H"),
+                    FieldLenField("len", None, length_of="fragment", fmt="!H"),
                     StrLenField("fragment", "",
-                                length_from = lambda pkt: pkt.length) ]
+                                length_from = lambda pkt: pkt.len) ]
+
+    def __getattribute__(self, attr):
```

Comment:
I am soooooo sorry. I did not understand the comment that you made yesterday: a packet field cannot have the same name as its methods. Renaming fragment to data was the way to go.

Let's see what @p-l- will decide. If he prefers to rename the field, I will provide a patch that you will apply to this PR. Don't do the job, I will!

Again, I am so sorry for this misunderstanding.

Reply:
NP. I am sorry for my 'english' as a reason of misunderstanding :(
PS: __get/set/delattribute__ is not bad but look like a gun against fly... :)

Reply:
> Don't do the job, I will!
I will! :) Why? Because I still have ready to use/appropriate branch.

Reply:
I'm not sure to get the point here, but I think I'd rather rename the `fragment` field...